pycross/private/tools/wheel_installer.py

```diff
@@ -39,6 +45,31 @@ def setup_namespace_pkg_compatibility(wheel_dir: Path) -> None:
         namespace_pkgs.add_pkgutil_style_namespace_pkg_init(ns_pkg_dir)
 
 
+class FilteredWheelFile(WheelFile):
+    def __init__(self, f: zipfile.ZipFile, install_exclude_globs: List[str]) -> None:
+        super().__init__(f)
+        self.install_exclude_globs = install_exclude_globs
+
+    @classmethod
+    @contextmanager
+    def open(cls, path: "os.PathLike[str]", install_exclude_globs: List[str]) -> Iterator["FilteredWheelFile"]:
```

Comment:
add
```
from __future__ import annotations

from typing import Union
```

and this can be
```
def open(cls, path: Union[os.PathLike, str], install_exclude_globs: List[str]) -> Iterator[FilteredWheelFile]:
```

Reply:
done